dist/tools/suit/suit-manifest-generator/README.md

```diff
@@ -144,6 +144,8 @@ The `suit-tool` supports three sub-commands:
 * `create` generates a new manifest.
 * `sign` signs a manifest.
 * `parse` parses an existing manifest into cbor-debug or a json representation.
+* `keygen` Create a signing key. Not for production use.
```

Comment:
Is that a private key? And why not for production use?

Reply:
Ahh, crap, this was not supposed to be reviewed, everything under dist/tools/suit/suit-manifest-generator is imported from ARMmbed/suit-manifest-generator and will be migrated to a package when the spec is stabilized.
I'd rather not start fixing nitpicks in "external" code and only fix the bare minimum required for integration.